src/adapter/bridges/ScrollERC20Bridge.ts

```diff
@@ -127,4 +129,11 @@ export class ScrollERC20Bridge extends BaseBridgeAdapter {
   protected getScrollGatewayRouter(): Contract {
     return this.scrollGatewayRouter;
   }
+
+  protected override resolveL2TokenAddress(l1Token: string): string {
+    if (compareAddressesSimple(TOKEN_SYMBOLS_MAP.USDC.addresses[this.hubChainId], l1Token)) {
+      return TOKEN_SYMBOLS_MAP.USDC.addresses[this.l2chainId]; // Scroll only has one USDC token type.
+    }
+    return super.resolveL2TokenAddress(l1Token);
```

Comment:
Good call. Thoughts on
```suggestion
  protected override resolveL2TokenAddress(l1Token: string): string {
  
    if () {
      return TOKEN_SYMBOLS_MAP.USDC.addresses[this.l2chainId]; // Scroll only has one USDC token type.
    }
    return super.resolveL2TokenAddress(l1Token);
```